src/poliastro/core/propagation/__init__.py

```diff
@@ -448,10 +412,12 @@ def pimienta(k, r0, v0, tof):
 
     # Solve first for eccentricity and mean anomaly
     p, ecc, inc, raan, argp, nu = rv2coe(k, r0, v0)
+    q = p / (1 + ecc)
+
+    # TODO: Do something to increase parabolic accuracy?
+    n = np.sqrt(k * (1 - ecc) ** 3 / q ** 3)
+    M0 = E_to_M(nu_to_E(nu, ecc), ecc)
 
-    M0 = nu_to_M(nu, ecc, delta=0)
```

Comment:
Interesting: we were not using the near-parabolic region. What changed, then?

Reply:
`E_to_nu` changed.

With this script:

```python
from poliastro.core.angles import E_to_nu


E = 2.0029132060900078e-07
ecc = 0.9999999999999996

nu = E_to_nu(E, ecc)
print(nu)
```

Before:

```
$ python e2nu.py
2.8461703827868337
```

After:

```
$ python e2nu.py
2.8466910769983373
```

Investigating.

Reply:
Checked with `mpmath`, both floating point implementations match their arbitrary precision result, but they are still different. Which formula is the correct one?

Reply:
The difference is less than 0.02 %.

Reply:
Changed to a formula that gave better results.